phys2bids/phys2bids.py

```diff
@@ -176,7 +176,7 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
 
 def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
               sub=None, ses=None, chtrig=0, chsel=None, num_timepoints_expected=0,
-              tr=1, thr=2.5, ch_name=[], chplot='', debug=False, quiet=False):
+              tr=1, thr=2.5, ch_name=[], chplot='', debug=False, quiet=False, std_option=0):
```

Comment:
I think in this case instead of adding an option we could activate the the previous function only if a threshold is given

Reply:
So delete the std_option and put an action in the ```-thr`` option that activates the if. What do you think?

Reply:
Mmm, I don't know. On one side, the user might want to run the program without automatic trigger count. On the other, we might have too many arguments, so deleting one is not a bad idea.    @rmarkello @eurunuela what do you think?

Reply:
We could make `thr=None` and run the automatic threshold in this case. If the user provides a threshold value, we don't run the automatic thing.

Reply:
I was looking at the code in the review. If we move the trigger detection step inside an `if` statement that checks that the user provided `ntp` and `tr`, we should be ok.
In this way, if the users want any trigger detection (manual or not), they can add the other two parameters, otherwise drop it.